circuit-construction/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
```

Comment:
As said for develop, I do not know if this is used.
Let's keep it, but we should see if we can remove it.
Also, in a follow-up PR, I would use the workspace everywhere in this Cargo.toml file.

Reply:
Will fix the workspace. This workspace was not unified in `master`, and `master` does not yet have the changes of `develop` where /all/ the workspaces are unified.

Reply:
Opened https://github.com/o1-labs/proof-systems/issues/2607